server/zclient.go

```diff
@@ -231,18 +231,18 @@ func newIPRouteBody(dst pathList) (body *zebra.IPRouteBody, isWithdraw bool) {
 	switch path.GetRouteFamily() {
 	case bgp.RF_IPv4_UC, bgp.RF_IPv4_VPN:
 		if path.GetRouteFamily() == bgp.RF_IPv4_UC {
-			prefix = path.GetNlri().(*bgp.IPAddrPrefix).IPAddrPrefixDefault.Prefix.To4()
+			prefix = path.GetNlri().(*bgp.IPAddrPrefix).Prefix.To4()
```

Comment:
This should be refactored to future proof the code from panics, even if currently this is highly unlikely, i.e.:

```Go
switch T := path.GetNlri().(type) {
case *bgp.IPAddrPrefix:
  prefix = T.Prefix.To4()
case *bgp.LabeledVPNIPAddrPrefix):
  ...
default:
  ... what is sane in this case? Right now that is a panic about concrete types, can we do better?
```